Source/SPDataImport.m

```diff
@@ -320,7 +320,7 @@ - (void)importFile
 		if (importFileName == nil) return;
 		
 		// Check to see if current connection has existing tables, if so warn
-		if([[tablesListInstance tables] count] > 1){
+		if([[tablesListInstance tables] count] > 1 && [[[importFormatPopup selectedItem] title] isEqualToString:@"SQL"]){
```

Comment:
What do you think about changing it to `![[[importFormatPopup selectedItem] title] isEqualToString:@"CSV"]`?

Reply:
SQL and CSV are currently the only supported formats so it wouldn't make much of a difference I guess. But SQL is the only destructive format I can think of that can alter existing data. So I guess it makes more sense the warning would only apply to SQL? The CSV format option also includes the tsv file extention btw.